### PR TITLE
add cases to test different auth when forwarding logs to fluentd

### PR DESCRIPTION
**Pull request by QiaolingTang**
Update cases to test different auth when forwarding logs via fluentdForward.
/cc @anpingli @gkarager

### CODE REVIEW

**Comment by QiaolingTang** [comment]
I saw the [fix for 4.6](https://bugzilla.redhat.com/show_bug.cgi?id=1929420) is ready to test now, so I think this PR can be ready now.

**Review by anpingli — COMMENTED** [review]
/lgtm

File: features/step_definitions/logging.rb [hunk 1]
```diff
@@ -535,10 +556,23 @@
...
+        step %Q/I create pipelinesecret with auth type mTLS/
+        configmap_file = "#{file_dir}/fluentd/secure/cm-mtls.yaml"
+      when "server_auth"
+        step %Q/I create pipelinesecret with auth type server_auth/
+        configmap_file = "#{file_dir}/fluentd/secure/cm-serverauth.yaml"
+      when "server_auth_share"
+        step %Q/I create pipelinesecret with auth type server_auth_share/
+        configmap_file = "#{file_dir}/fluentd/secure/cm-serverauth-share.yaml"
+      else
+        raise "Unrecognized auth type: #{auth_type}"
```
Comment (liangxia) [inline features/step_definitions/logging.rb:574]:
```
      case auth_type
      when "mTLS_share"
        configmap_file = "#{file_dir}/fluentd/secure/cm-mtls-share.yaml"
      when "mTLS"
        configmap_file = "#{file_dir}/fluentd/secure/cm-mtls.yaml"
      when "server_auth"
        configmap_file = "#{file_dir}/fluentd/secure/cm-serverauth.yaml"
      when "server_auth_share"
        configmap_file = "#{file_dir}/fluentd/secure/cm-serverauth-share.yaml"
      else
        raise "Unrecognized auth type: #{auth_type}"
      end
      step %Q/I create pipelinesecret with auth type #{auth_type}/
```

Comment (QiaolingTang) [reply features/step_definitions/logging.rb:570]: Updated, could you please take a look again?

**Comment by QiaolingTang** [comment]
Runner-v3 log: https://url.corp.redhat.com/b5fb3a3

**Pull request merged**